### ISSUE TITLE
Recurring ASG schedule syntax error

### ISSUE THREAD

**Issue by revett** [hashicorp/terraform#4628]
> **Tested on v0.6.9**

### TF

```
# ASG

resource "aws_autoscaling_group" "asg_main" {
  depends_on = ["aws_launch_configuration.asg_launch_config", "aws_elb.elb_main"]
  name = "asg_main"

  launch_configuration = "${aws_launch_configuration.asg_launch_config.id}"
  load_balancers = ["${aws_elb.elb_main.name}"]

  vpc_zone_identifier = ["${split(",", var.public_subnets)}"]

  min_size = "${var.asg_min_size}"
  max_size = "${var.asg_max_size}"
  desired_capacity = "${var.asg_desired_size}"

  health_check_grace_period = 300
  health_check_type = "EC2"

  termination_policies = ["OldestLaunchConfiguration", "ClosestToNextInstanceHour"]

  lifecycle {
    create_before_destroy = true
  }

  tag {
    key = "tutum:app"
    value = "${var.tag_tutum_app}"
    propagate_at_launch = true
  }
  tag {
    key = "tutum:env"
    value = "${var.tag_tutum_env}"
    propagate_at_launch = true
  }
}

# ASG Scheduling

resource "aws_autoscaling_schedule" "asg_schedule_morning" {
  depends_on = ["aws_autoscaling_group.asg_main"]
  scheduled_action_name = "${var.env}_asg_schedule_morning"
  desired_capacity = "${var.asg_desired_size}"
  recurrence = "0 8 * * *"
  autoscaling_group_name = "${aws_autoscaling_group.asg_main.name}"
}

resource "aws_autoscaling_schedule" "asg_schedule_evening" {
  depends_on = ["aws_autoscaling_group.asg_main"]
  scheduled_action_name = "${var.env}_asg_schedule_evening"
  desired_capacity = 0
  recurrence = "0 23 * * *"
  autoscaling_group_name = "${aws_autoscaling_group.asg_main.name}"
}
```

### Problem

I added two `aws_autoscaling_schedule` resources and got the following error:

```
* aws_autoscaling_schedule.asg_schedule_morning: Error Creating Autoscaling Scheduled Action: ValidationError: Scheduled start time must be specified for non-recurrent future group action
```

Why is a `start_time` required if I have already outlined the timing with cron syntax?

Thanks - C

**Comment by stack72** [comment]
@revett I believe this was fixed in https://github.com/hashicorp/terraform/pull/4579

I will get the acceptance tests updated for this

Sorry!

P.

**Comment by revett** [comment]
@stack72 thanks!

**Comment by jen20** [comment]
This should be fixed by #4674. Thanks for reporting @revett!

**Comment by revett** [comment]
@jen20 no worries - when will this be released?
